cdap-common/bin/functions.sh

```diff
@@ -128,6 +128,8 @@ cdap_create_log_dir() { test -d "${LOG_DIR}" || mkdir -p "${LOG_DIR}"; };
 #
 cdap_create_pid_dir() { test -d "${PID_DIR}" || mkdir -p "${PID_DIR}"; };
 
+cdap_create_dir() { test -d "$1" || mkdir -p "$1"; }
```

Comment:
Is that just better practice?
I found "$1" usage here: http://tldp.org/LDP/abs/html/complexfunct.html